src/types/DataTypes.sol

```diff
@@ -192,6 +184,14 @@ library LockupDynamic {
         Lockup.Amounts amounts;
         Segment[] segments;
     }
+
+    /// @notice Struct encapsulating the LockupDynamic timestamps.
+    /// @param start The Unix timestamp indicating the stream's start.
+    /// @param end The Unix timestamp indicating the stream's end.
+    struct Timestamp {
```

Comment:
Timestamps

Reply:
The original idea was to rename `Range` to Timestamp` and use plural, `timestamps` for variable name. Ref: https://github.com/sablier-labs/v2-core/discussions/902#discussioncomment-9118755

Reply:
I am little bit confused by these:

1.
<img width="492" alt="image" src="https://github.com/sablier-labs/v2-core/assets/99738872/89af6979-b0d7-44e9-82b4-6f8c6e0d42c7">

2. 
<img width="432" alt="image" src="https://github.com/sablier-labs/v2-core/assets/99738872/71e63b9c-d2ba-48fc-9be5-b17c10be0fa5">

IMO, it does not make sense to have `Timestamp` over `Timestamps` since there are multiple time variables within the structs, and the variable name is `timestamps`. And one other argument is that we have the function`createWithTimestamps`.

Reply:
I agree with your arguments. I would prefer `Timestamps` over `Timestamp`.

Reply:
@PaulRBerg could you please confirm, which version you suggested?

Reply:
It is true that originally I/ we said `Timestamp`, but looking at the PR code I realized that `Timestamps` would work much better.

Reply:
Thank you for agreeing.